src/secrets.template.ts

```diff
@@ -1,10 +1,10 @@
-export const accounts: [string, string][] = [
-	["USERNAME", "PASSWORD"], // maid's party night
-	["USERNAME", "PASSWORD"], // maid's party night
-	["USERNAME", "PASSWORD"], // gameroom
-	["USERNAME", "PASSWORD"], // gameroom2
-	["USERNAME", "PASSWORD"], // gameroom3
-	["USERNAME", "PASSWORD"]  // gameroom4
-];
+export const accounts: Record<string, ([string, string] | [string, string][])> = {
+	"maidspartynight": [["USERNAME", "PASSWORD"], ["USERNAME", "PASSWORD"]],
+	"iwouldnotmind": ["USERNAME", "PASSWORD"],
+	"roleplaychallenge": ["USERNAME", "PASSWORD"],
+	"kidnappers": ["USERNAME", "PASSWORD"],
+	"gameroom4": ["USERNAME", "PASSWORD"],
+	"magicstrangeroom": ["USERNAME", "PASSWORD"]
+};
```

Comment:
You can instead of specifying type mark it as `as const` - that will let you drop all the type assertions everywhere
```suggestion
export const accounts = {
	"maidspartynight": [["USERNAME", "PASSWORD"], ["USERNAME", "PASSWORD"]],
	"iwouldnotmind": ["USERNAME", "PASSWORD"],
	"roleplaychallenge": ["USERNAME", "PASSWORD"],
	"kidnappers": ["USERNAME", "PASSWORD"],
	"gameroom4": ["USERNAME", "PASSWORD"],
	"magicstrangeroom": ["USERNAME", "PASSWORD"]
} as const;
```

Reply:
👍